spyderlib/plugins/editor.py

```diff
@@ -944,7 +952,7 @@ def get_plugin_actions(self):
         self.recent_file_menu = QMenu(_("Open &recent"), self)
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 

```

Comment:
I made the necessary changes, please go ahead and merge! This is my first code PR to spyder :smile: